### DISCUSSION TITLE
Change Row Color based on Column value

### DISCUSSION THREAD

**Discussion by ollih1980** [AvaloniaUI/Avalonia#7116] [Q&A]
Hey Community,

I would like to change the row color based on a value in a DataGrid Column.
But I simply don't can make it work.

this is what I've got:

```
<DataGrid Name="MyDataGrid" Items="{Binding}">
	<i:Interaction.Behaviors>
		<ia:DataTriggerBehavior Binding="{Binding Status}" ComparisonCondition="Equal" Value="SOMEVALUE">
			<ia:ChangePropertyAction TargetObject="{Binding #MyDataGrid}" PropertyName="Background" Value="DarkOrange" />
		</ia:DataTriggerBehavior>
	</i:Interaction.Behaviors>
</DataGrid>
```

Could somebody help me? Thx.

**Comment by maxkatz6** [comment]
Use event LoadingRow

**Reply by ollih1980** [reply]
But when I change the content of a Cell, the event will not fire... So what I want to achieve is something like this:

1. Select a Row
2. Push a button
3. Color the selected row in a specific color

**Reply by maxkatz6** [reply]
Inside of LoadinfRow you can access both Row (including it's cells) and data context. So you can add binding from the datacontext on some attached property or style-class of the row.

**Reply by maxkatz6** [reply]
Assuming that pushing the button will update this data context

**Reply by ollih1980** [reply]
Could you pls give some more details? How to do this binding? Any code example?

**Reply by ollih1980** [reply]
Any example available?
